packages/react-microlink/src/components/Card/CardMedia/image.js

```diff
@@ -1,6 +1,8 @@
 import MediaWrap from './wrap'
 
-export default MediaWrap.extend`
+export default MediaWrap.extend.attrs({
+  className: 'microlink_card__media_image'
+})`
   background: no-repeat center center / cover;
-  ${({image}) => image && `background-image: url(${image});`}
+  background-image: ${({image}) => `url('${image}')` || ''};
```

Comment:
Why this line change?

Reply:
Just to follow the convention of how we've used prop styles, just noticed I can actually make a small change to it...